test/e2e/setup.sh

```diff
@@ -104,3 +104,6 @@ do
         --set enableV1Alpha1APIs=false \
         --set enableV1Beta1APIs=true
 done
+
+# Download CRD from Fleet networking repo
+curl "https://github.com/Azure/fleet-networking/blob/v0.2.7/config/crd/bases/networking.fleet.azure.com_internalserviceexports.yaml" | jq -r '.payload.blob.rawLines' | jq -r '.[]' > ./internalserviceexport-crd.yaml
```

Comment:
Hi Arvind! I think the better way is just to use 

```
export NETWORKING_CRD_URL=https://raw.githubusercontent.com/Azure/fleet-networking/v0.2.7/config/crd/bases/networking.fleet.azure.com_internalserviceexports.yaml
curl $NETWORKING_CRD_URL | kubectl apply -f -
```

You do not have save the file actually. And it saves the trouble of having to clean things up or installing them in the code.

Reply:
This can be added after line 73 I believe.